main.py

```diff
@@ -121,20 +125,21 @@ def add_admins(message: Message):
             msg = bot.send_message(message.chat.id, "You're not a moderator❌")
         autodelete(bot, msg, 2)
         autodelete(bot, message, 1)
+    else:
+        bot.send_message(message.chat.id, parse_mode="HTML", text="<b>Admins can be added ONLY in our group,\n\
```

Comment:
Every `parse_mode="HTML"` are useless because already defined in bot object

Reply:
Oops, my bad :)